roles/openshift_node/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-- name: restart openshift-node
-  service: name=openshift-node state=restarted
+- name: restart node
+  service: name={{ product_type }}-node state=restarted
```

Comment:
@ashcrow we'd still need to differentiate some things between atomic enterprise and openshift (I'm thinking app templates and other thinks for openshift that wouldn't apply to atomic enterprise).

We have 2 different things we need to key off of the way I see it.
1. binary/service names - deployment_type (origin vs online/enterprise)
2. product features - product_type (openshift vs atomic-enterprise)